flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,436 @@
+#     log.verbose('#####################################################################################')
+
+
+class AnidbFeed(object):
+    """"Creates an entry for each movie or series in the AniDB notification feed.
+        Your personalized notification feed url is shown under [Account/Settings/Notifications/link to your personal atom feed]
+        See: https://wiki.anidb.net/w/Notifications
+        The general main anidb feed can also be used url: http://anidb.net/feeds/files.atom
+
+        Example:
+            anidb_feed:
+                url: http://anidb.net/perl-bin/animedbfeed.pl?id=xxxxx................
+                valid_only: yes
+
+            # valid_only: yes
+            # If enabled only 'valid' entries are returned, that have a size, release group, resolution and source field
+            # This happens after the file was added to anidb and has passed some initial tests/parsing
+
+        Adds those extra Entry fields if found:
+        'content_size'
+        'rss_pubdate'
+        'anidb_fileversion'     # fileversion aka v1-5
+        'anidb_fid'             # anidb file_id
+        'anidb_feed_added'
+        'anidb_feed_source'
+        'anidb_feed_resolution'
+        'anidb_feed_crc_status'
+        'anidb_feed_language'
+        'anidb_feed_group'
+        'anidb_feed_subtitle_language'
+        'anidb_feed_priority'
+        'anidb_feed_quality'
+        'anidb_feed_size'
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'url': {'type': 'string', 'format': 'url'},
+            'priority': {'type': 'string', 'enum': ['all', 'medium-high', 'high'], 'default': 'all'},
+            'valid_only': {'type': 'boolean', 'default': False},
+        },
+        'additionalProperties': False,
+        'required': ['url'],
+    }
+
+    # notification update interval is 15 minutes
+    #@cached('anidb_feed', persist='15 minutes')
+    def on_task_input(self, task, config):
+        # Create entries by parsing AniDB feed
+        log.verbose('Retrieving AniDB feed')
+        url = re.sub(r'&pri=[1-9]', '', config['url'])  # strip existing priority
+        priority_fragment = ''
+        if 'animedbfeed' in url:
+            priority = 0
+            if config['priority'] is 'medium-high':
+                priority = 1
+            elif config['priority'] is 'high':
+                priority = 2
+            priority_fragment = '&pri=%s' % priority
+        return self.fill_entries_for_url(url + priority_fragment, task, config)
+
+    @staticmethod
+    def is_valid_entry(entry, ns_prefix_name=''):
+        for key in VALIDATION_LIST:
+            key_name = ns_prefix_name + key
+            if not entry.get(key_name):  # int(0) is invalid
+                return False
+            elif entry[key_name] in ['Raw/Unknown', 'N/A', 'Unchecked']:
+                return False
+        return True
+
+    def parse_from_xml(self, xml_entries, config):
```

Comment:
You should try to split this, too long method. Some say correct size is about 10 lines :)